python/dask_cudf/dask_cudf/core.py

```diff
@@ -65,7 +67,7 @@ class _Frame(dd.core._Frame, OperatorMethodMixin):
         Values along which we partition our blocks on the index
     """
 
-    __dask_scheduler__ = staticmethod(dask.get)
+    __dask_scheduler__ = staticmethod(scheduler.get_sync)
```

Comment:
Is this just for debugging purposes to make this synchronous?

Reply:
Oops yes, I will fix that.